src/hooks/useTokenMaps.ts

```diff
@@ -0,0 +1,43 @@
+import { BasicTokensMap, PoolName, TokenToPoolsMap } from "../constants"
+
+import { BasicPoolsContext } from "../providers/BasicPoolsProvider"
+import { TokensContext } from "../providers/TokensProvider"
+import { useContext } from "react"
+
+export const useTokenMaps = (): {
```

Comment:
Add a comment somewhere that this maps token symbols to various things, so devs don't try key on addresses or something else

Reply:
@ug02fast  Bumping this. Make sure you have a comment here addressing this.